latest/index.bs

```diff
@@ -315,10 +315,11 @@ Conforming groups:
 Additionally:
 
 - If "OME/METADATA.ome.xml" is present, "OME" MUST be a Zarr group which:
-    - MAY contain a "series" attribute.
-    - "series" MUST be a list of string objects, each of which is a path to an image group.
-    - The order of the paths MUST match the order of the "Image" elements in "OME/METADATA.ome.xml".
+    - MAY contain a "series" attribute. If so:
+        - "series" MUST be a list of string objects, each of which is a path to an image group.
+        - The order of the paths MUST match the order of the "Image" elements in "OME/METADATA.ome.xml".
 - If "OME/METADATA.ome.xml" or the "series" attribute do not exist:
+    - existing "plate" metadata will take precedence if it exists, or
```

Comment:
Does `will` here actually mean `MUST`?

Reply:
Hmmm.... I _think_ that's right, but what I was trying to get across is that "the next statement only holds when `"plate"` isn't defined". Is that the same thing as the MUST?

Reply:
I think what bothers me in the logic is that the "plate MUST take precedence" applies much higher, e.g. even if the XML exists. Let me try to break all the clauses apart.

Reply:
Primary question from my side is if it's actually:

```
Matching "series" metadata (as described next) SHOULD be provided ...
```

Reply:
Latest version is definitely clearer, thank you. I think `Matching "series" metadata (as described next) SHOULD be provided` makes sense, but I don't have strong objections to `MAY`.

Reply:
If we're unsure, I'll err on the side of the stricter `SHOULD` and then we can downgrade it later.